[2020-02-05 10:03:02] WARNING [django.request.get_response:93] Not Found: /robots.txt
[2020-02-05 10:03:02] WARNING [django.channels.server.log_action:149] HTTP GET /robots.txt 404 [0.02, 127.0.0.1:2128]
[2020-02-05 10:03:02] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.61, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.07, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.05, 127.0.0.1:2128]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.05, 127.0.0.1:2137]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.05, 127.0.0.1:2140]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.05, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.08, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.10, 127.0.0.1:2139]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.08, 127.0.0.1:2128]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.07, 127.0.0.1:2137]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.04, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.07, 127.0.0.1:2140]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.06, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.03, 127.0.0.1:2137]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.02, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.03, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.02, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.02, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.02, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.03, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.03, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.02, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.02, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.03, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.02, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.03, 127.0.0.1:2127]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.03, 127.0.0.1:2138]
[2020-02-05 10:03:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.02, 127.0.0.1:2127]
[2020-02-05 10:03:04] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.06, 127.0.0.1:2127]
[2020-02-05 10:03:04] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.07, 127.0.0.1:2138]
[2020-02-05 10:03:06] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.01, 127.0.0.1:2138]
[2020-02-05 10:07:05] INFO [django.channels.server.log_action:141] HTTP GET /signup 200 [0.34, 127.0.0.1:2296]
[2020-02-05 10:07:05] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/styles.css 200 [0.02, 127.0.0.1:2296]
[2020-02-05 10:07:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/imgback.jpg 200 [0.08, 127.0.0.1:2296]
[2020-02-05 10:07:07] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-05 10:07:07] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.04, 127.0.0.1:2296]
[2020-02-05 10:07:24] INFO [django.channels.server.log_action:141] HTTP POST /signup 200 [0.44, 127.0.0.1:2296]
[2020-02-05 10:07:24] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/checkers.png 200 [0.02, 127.0.0.1:2296]
[2020-02-05 10:07:25] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/signupbg.png 200 [0.06, 127.0.0.1:2296]
[2020-02-05 10:08:49] INFO [django.channels.server.log_action:147] HTTP GET /admin/ 302 [0.11, 127.0.0.1:1096]
[2020-02-05 10:08:49] INFO [django.channels.server.log_action:141] HTTP GET /admin/login/?next=/admin/ 200 [0.06, 127.0.0.1:1096]
[2020-02-05 10:08:50] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/css/responsive.css 304 [0.01, 127.0.0.1:1105]
[2020-02-05 10:08:50] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/login.css 200 [0.35, 127.0.0.1:1104]
[2020-02-05 10:08:50] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/base.css 200 [0.36, 127.0.0.1:1096]
[2020-02-05 10:08:50] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/css/fonts.css 304 [0.01, 127.0.0.1:1104]
[2020-02-05 10:08:53] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Light-webfont.woff 200 [0.06, 127.0.0.1:1105]
[2020-02-05 10:08:53] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Regular-webfont.woff 200 [0.07, 127.0.0.1:1104]
[2020-02-05 10:08:53] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-05 10:08:53] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.02, 127.0.0.1:1105]
[2020-02-05 10:09:06] INFO [django.channels.server.log_action:147] HTTP POST /admin/login/?next=/admin/ 302 [0.32, 127.0.0.1:1105]
[2020-02-05 10:09:06] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.25, 127.0.0.1:1105]
[2020-02-05 10:09:06] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/css/base.css 304 [0.01, 127.0.0.1:1105]
[2020-02-05 10:09:06] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/css/dashboard.css 304 [0.01, 127.0.0.1:1104]
[2020-02-05 10:09:09] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/img/icon-addlink.svg 304 [0.03, 127.0.0.1:1104]
[2020-02-05 10:09:09] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/img/icon-changelink.svg 304 [0.04, 127.0.0.1:1096]
[2020-02-05 10:09:09] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/fonts/Roboto-Bold-webfont.woff 304 [0.04, 127.0.0.1:1144]
[2020-02-05 10:09:09] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-05 10:09:09] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.14, 127.0.0.1:1105]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/emailmessagelog/ 200 [0.73, 127.0.0.1:1300]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/css/changelists.css 304 [0.01, 127.0.0.1:1300]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/vendor/jquery/jquery.js 304 [0.02, 127.0.0.1:1311]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/jquery.init.js 304 [0.03, 127.0.0.1:1312]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/core.js 304 [0.03, 127.0.0.1:1313]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/actions.js 304 [0.03, 127.0.0.1:1300]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/admin/RelatedObjectLookups.js 304 [0.03, 127.0.0.1:1314]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/urlify.js 304 [0.03, 127.0.0.1:1311]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.07, 127.0.0.1:1310]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/vendor/xregexp/xregexp.js 304 [0.03, 127.0.0.1:1313]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:145] HTTP GET /static/admin/js/prepopulate.js 304 [0.03, 127.0.0.1:1312]
[2020-02-05 10:13:51] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/assets/vendor/bootstrap/css/bootstrap.min.css 200 [0.05, 127.0.0.1:1300]
[2020-02-05 10:13:54] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-no.svg 200 [0.01, 127.0.0.1:1310]
[2020-02-05 10:13:54] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/tooltag-add.svg 200 [0.01, 127.0.0.1:1310]
[2020-02-05 10:13:54] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-yes.svg 200 [0.06, 127.0.0.1:1314]
[2020-02-05 10:27:50] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.53, 127.0.0.1:2038]
[2020-02-05 10:27:50] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.03, 127.0.0.1:2038]
[2020-02-05 10:27:50] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.01, 127.0.0.1:2038]
[2020-02-05 10:27:54] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.11, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/ 200 [0.20, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/we_new.css 200 [0.02, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/workexperience.css 200 [0.02, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/arrow-right.png 200 [0.04, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/cup.png 200 [0.05, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/tap.png 200 [0.02, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/number.png 200 [0.04, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/accept.png 200 [0.05, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/arrow-blue.png 200 [0.01, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/join.png 200 [0.04, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/graph.svg 200 [0.03, 127.0.0.1:2053]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/list%20_checked.png 200 [0.03, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/cap.png 200 [0.02, 127.0.0.1:2057]
[2020-02-05 10:27:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/wallet.png 200 [0.03, 127.0.0.1:2053]
[2020-02-05 10:28:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/el.png 200 [0.04, 127.0.0.1:2057]
[2020-02-05 10:28:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/bg2.png 200 [0.05, 127.0.0.1:2053]
[2020-02-05 10:28:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/alumni-bg.png 200 [0.05, 127.0.0.1:2063]
[2020-02-05 10:28:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/thmbs.png 200 [0.04, 127.0.0.1:2067]
[2020-02-05 10:28:06] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/terms/ 200 [0.19, 127.0.0.1:2077]
[2020-02-05 10:28:10] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/pay/ 200 [0.15, 127.0.0.1:2077]
[2020-02-05 10:28:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/linuxlogo.png 200 [0.04, 127.0.0.1:2113]
[2020-02-05 10:28:32] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/pay/ 302 [1.75, 127.0.0.1:2123]
[2020-02-05 10:28:33] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/eligibility/ 200 [0.23, 127.0.0.1:2123]
[2020-02-05 10:33:13] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/images/ui-icons_444444_256x240.png 200 [0.06, 127.0.0.1:2385]
[2020-02-05 10:33:44] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /workexperience/eligibility/
Traceback (most recent call last):
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 1397, in work_experience_eligible
    det = WorkExperienceEligibility.objects.get(user=request.user)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\db\models\query.py", line 403, in get
    self.model._meta.object_name
home.models.DoesNotExist: WorkExperienceEligibility matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 1426, in work_experience_eligible
    work_experience_eligible_pdf(request.user)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 1284, in work_experience_eligible_pdf
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-05 10:33:44] ERROR [django.channels.server.log_action:154] HTTP POST /workexperience/eligibility/ 500 [12.25, 127.0.0.1:2405]
[2020-02-05 10:33:55] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/eligibility/ 200 [0.22, 127.0.0.1:2435]
[2020-02-05 10:34:12] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /workexperience/eligibility/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 1420, in work_experience_eligible
    work_experience_eligible_pdf(det.user)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 1284, in work_experience_eligible_pdf
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-05 10:34:12] ERROR [django.channels.server.log_action:154] HTTP POST /workexperience/eligibility/ 500 [12.10, 127.0.0.1:2435]
[2020-02-05 10:35:16] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/eligibility/ 200 [0.16, 127.0.0.1:2537]
[2020-02-05 10:35:29] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/eligibility/ 302 [7.34, 127.0.0.1:2556]
[2020-02-05 10:35:29] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/ISA/ 200 [0.21, 127.0.0.1:2556]
[2020-02-05 11:48:40] INFO [django.channels.server.log_action:147] HTTP GET /noobaid 301 [0.03, 127.0.0.1:2815]
[2020-02-05 11:48:40] INFO [django.channels.server.log_action:147] HTTP GET /noobaid/ 302 [0.08, 127.0.0.1:2815]
[2020-02-05 11:48:40] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.11, 127.0.0.1:2815]
[2020-02-05 11:48:42] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/proj-list.png 200 [0.10, 127.0.0.1:2815]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.27, 127.0.0.1:2837]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /noobaid/ 200 [0.08, 127.0.0.1:2837]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/noobaid.png 200 [0.04, 127.0.0.1:2837]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ilus1.png 200 [0.04, 127.0.0.1:2838]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/noobillus2.png 200 [0.01, 127.0.0.1:2837]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/tv.png 200 [0.01, 127.0.0.1:2838]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/pad.png 200 [0.04, 127.0.0.1:2838]
[2020-02-05 11:48:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/hap.png 200 [0.04, 127.0.0.1:2837]
[2020-02-05 11:48:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/noobanner.png 200 [0.07, 127.0.0.1:2838]
[2020-02-05 11:48:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/nav.png 200 [0.08, 127.0.0.1:2837]
[2020-02-05 11:48:49] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon2.png 200 [0.03, 127.0.0.1:2837]
[2020-02-05 11:59:25] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.17, 127.0.0.1:1039]
[2020-02-05 11:59:28] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.02, 127.0.0.1:1039]
[2020-02-05 12:00:01] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /internships
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\views.py", line 357, in internships
    internsh = InternshipDetail.objects.all()[0].date
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\db\models\query.py", line 307, in __getitem__
    return qs._result_cache[0]
IndexError: list index out of range
[2020-02-05 12:00:01] ERROR [django.channels.server.log_action:154] HTTP GET /internships 500 [0.12, 127.0.0.1:1039]
[2020-02-06 12:03:30] INFO [django.channels.server.log_action:147] HTTP GET / 302 [1.15, 127.0.0.1:1323]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.39, 127.0.0.1:1343]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.03, 127.0.0.1:1343]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.05, 127.0.0.1:1348]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.05, 127.0.0.1:1351]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.07, 127.0.0.1:1349]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.11, 127.0.0.1:1350]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.11, 127.0.0.1:1352]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.11, 127.0.0.1:1343]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.04, 127.0.0.1:1349]
[2020-02-06 12:03:32] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.10, 127.0.0.1:1348]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.05, 127.0.0.1:1352]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.14, 127.0.0.1:1351]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.12, 127.0.0.1:1350]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.03, 127.0.0.1:1352]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.04, 127.0.0.1:1351]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.03, 127.0.0.1:1351]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.04, 127.0.0.1:1352]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.03, 127.0.0.1:1351]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.04, 127.0.0.1:1352]
[2020-02-06 12:03:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.02, 127.0.0.1:1351]
[2020-02-06 12:03:34] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/proj-list.png 200 [0.06, 127.0.0.1:1351]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.26, 127.0.0.1:1351]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.19, 127.0.0.1:1351]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.02, 127.0.0.1:1351]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.04, 127.0.0.1:1352]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.05, 127.0.0.1:1350]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.04, 127.0.0.1:1350]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.04, 127.0.0.1:1352]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.01, 127.0.0.1:1352]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.01, 127.0.0.1:1350]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.02, 127.0.0.1:1352]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.04, 127.0.0.1:1350]
[2020-02-06 12:03:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.02, 127.0.0.1:1352]
[2020-02-06 12:03:42] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.08, 127.0.0.1:1352]
[2020-02-06 12:03:42] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.09, 127.0.0.1:1350]
[2020-02-06 12:03:42] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.07, 127.0.0.1:1350]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:147] HTTP GET /noobaid 301 [0.01, 127.0.0.1:1350]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /noobaid/ 200 [0.12, 127.0.0.1:1350]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/noobaid.png 200 [0.03, 127.0.0.1:1350]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ilus1.png 200 [0.03, 127.0.0.1:1352]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/noobillus2.png 200 [0.02, 127.0.0.1:1352]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/tv.png 200 [0.02, 127.0.0.1:1350]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/pad.png 200 [0.03, 127.0.0.1:1352]
[2020-02-06 12:03:48] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/hap.png 200 [0.03, 127.0.0.1:1350]
[2020-02-06 12:03:50] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/noobanner.png 200 [0.06, 127.0.0.1:1352]
[2020-02-06 12:03:50] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/nav.png 200 [0.07, 127.0.0.1:1350]
[2020-02-06 19:12:28] INFO [django.channels.server.log_action:147] HTTP GET / 302 [0.46, 127.0.0.1:2138]
[2020-02-06 19:12:28] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.21, 127.0.0.1:2138]
[2020-02-06 19:12:34] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.25, 127.0.0.1:2138]
[2020-02-06 19:12:34] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.14, 127.0.0.1:2138]
[2020-02-06 19:12:37] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.08, 127.0.0.1:2138]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /it_partnership/ 200 [0.09, 127.0.0.1:2177]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/it-partner.css 200 [0.06, 127.0.0.1:2177]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/Small-final.png 200 [0.05, 127.0.0.1:2178]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/group-member.png 200 [0.08, 127.0.0.1:2184]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/pointer.png 200 [0.02, 127.0.0.1:2178]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/startup.png 200 [0.05, 127.0.0.1:2184]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/card-vector.png 200 [0.06, 127.0.0.1:2178]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/code-icon.png 200 [0.02, 127.0.0.1:2184]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/server-icon.png 200 [0.02, 127.0.0.1:2178]
[2020-02-06 19:12:40] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/cloud-icon.png 200 [0.02, 127.0.0.1:2184]
[2020-02-06 19:12:42] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/service-item-bg.png 200 [0.01, 127.0.0.1:2184]
[2020-02-11 08:35:46] WARNING [django.request.get_response:93] Not Found: /robots.txt
[2020-02-11 08:35:46] WARNING [django.channels.server.log_action:149] HTTP GET /robots.txt 404 [0.04, 127.0.0.1:2475]
[2020-02-11 08:35:46] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.71, 127.0.0.1:2474]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.06, 127.0.0.1:2474]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.04, 127.0.0.1:2475]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.05, 127.0.0.1:2478]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.06, 127.0.0.1:2479]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.07, 127.0.0.1:2480]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.07, 127.0.0.1:2481]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.08, 127.0.0.1:2474]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.06, 127.0.0.1:2475]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.06, 127.0.0.1:2478]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.06, 127.0.0.1:2479]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.05, 127.0.0.1:2481]
[2020-02-11 08:35:47] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.05, 127.0.0.1:2480]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.05, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.06, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.03, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.03, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.04, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.04, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.02, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.03, 127.0.0.1:2504]
[2020-02-11 08:35:52] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:35:55] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.27, 127.0.0.1:2503]
[2020-02-11 08:35:55] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.06, 127.0.0.1:2503]
[2020-02-11 08:35:55] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.07, 127.0.0.1:2504]
[2020-02-11 08:35:57] INFO [django.channels.server.log_action:141] HTTP GET /login/ 200 [0.16, 127.0.0.1:2504]
[2020-02-11 08:35:57] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/styles.css 200 [0.02, 127.0.0.1:2504]
[2020-02-11 08:36:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/imgback.jpg 200 [0.08, 127.0.0.1:2504]
[2020-02-11 08:36:14] INFO [django.channels.server.log_action:147] HTTP POST /login/ 302 [0.66, 127.0.0.1:2504]
[2020-02-11 08:36:14] INFO [django.channels.server.log_action:141] HTTP GET /courses/userinterest 200 [0.13, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.02, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/lin-logo.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/button-left.png 200 [0.03, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/amazons.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ppt.png 200 [0.01, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/jjava.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/rchsaa.png 200 [0.02, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ajs.png 200 [0.02, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/csecurity.png 200 [0.01, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/biz.png 200 [0.02, 127.0.0.1:2503]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/dba.png 200 [0.03, 127.0.0.1:2504]
[2020-02-11 08:36:15] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/py.png 200 [0.03, 127.0.0.1:2503]
[2020-02-11 08:36:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs//list-bg.png 200 [0.07, 127.0.0.1:2503]
[2020-02-11 08:36:21] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.05, 127.0.0.1:2646]
[2020-02-11 08:36:55] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.07, 127.0.0.1:2672]
[2020-02-11 08:37:51] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/profile/pdf 301 [0.13, 127.0.0.1:2704]
[2020-02-11 08:37:51] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.11, 127.0.0.1:2704]
[2020-02-11 08:37:53] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-11 08:37:53] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.06, 127.0.0.1:2704]
[2020-02-11 08:41:23] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.09, 127.0.0.1:2785]
[2020-02-11 08:42:16] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.08, 127.0.0.1:2785]
[2020-02-11 08:42:43] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.09, 127.0.0.1:2785]
[2020-02-11 08:46:30] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.43, 127.0.0.1:2987]
[2020-02-11 08:52:51] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.11, 127.0.0.1:3454]
[2020-02-11 08:53:04] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.14, 127.0.0.1:3454]
[2020-02-11 08:53:08] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.07, 127.0.0.1:3454]
[2020-02-11 08:53:10] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-11 08:53:10] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.08, 127.0.0.1:3454]
[2020-02-11 08:53:35] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.10, 127.0.0.1:3454]
[2020-02-11 08:53:57] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.14, 127.0.0.1:3454]
[2020-02-11 08:54:05] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.13, 127.0.0.1:3454]
[2020-02-11 08:54:15] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.08, 127.0.0.1:3454]
[2020-02-11 08:54:45] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.20, 127.0.0.1:3454]
[2020-02-11 08:57:01] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/pdf/ 200 [0.10, 127.0.0.1:3877]
[2020-02-11 08:57:49] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.59, 127.0.0.1:4021]
[2020-02-11 08:57:55] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.03, 127.0.0.1:4022]
[2020-02-11 09:02:44] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [291.67, 127.0.0.1:4021]
[2020-02-11 09:02:47] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/IMG-20191210-WA0011_2.jpg 200 [0.17, 127.0.0.1:4021]
[2020-02-11 09:02:54] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/images/ui-icons_444444_256x240.png 200 [0.15, 127.0.0.1:4021]
[2020-02-11 09:02:54] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_UBCkgTw.pdf 200 [0.10, 127.0.0.1:7708]
[2020-02-11 09:02:58] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_Y62Zpn6.pdf 200 [0.10, 127.0.0.1:7708]
[2020-02-11 09:08:01] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [289.79, 127.0.0.1:7708]
[2020-02-12 11:31:20] WARNING [django.request.get_response:93] Not Found: /robots.txt
[2020-02-12 11:31:20] WARNING [django.channels.server.log_action:149] HTTP GET /robots.txt 404 [0.05, 127.0.0.1:1752]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.67, 127.0.0.1:1751]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.07, 127.0.0.1:1751]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.07, 127.0.0.1:1752]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.08, 127.0.0.1:1755]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.07, 127.0.0.1:1757]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.09, 127.0.0.1:1756]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.05, 127.0.0.1:1751]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.10, 127.0.0.1:1758]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.07, 127.0.0.1:1752]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.05, 127.0.0.1:1755]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.06, 127.0.0.1:1757]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.04, 127.0.0.1:1751]
[2020-02-12 11:31:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.04, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.06, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.07, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.02, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.03, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.02, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.02, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.02, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.02, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.02, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.01, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.01, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.02, 127.0.0.1:1756]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.02, 127.0.0.1:1751]
[2020-02-12 11:31:33] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.01, 127.0.0.1:1751]
[2020-02-12 11:31:37] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.03, 127.0.0.1:1751]
[2020-02-12 11:31:46] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.06, 127.0.0.1:1751]
[2020-02-12 11:31:46] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.07, 127.0.0.1:1756]
[2020-02-12 11:31:51] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.01, 127.0.0.1:1756]
[2020-02-12 11:33:16] INFO [django.channels.server.log_action:141] HTTP GET /login/ 200 [0.15, 127.0.0.1:1832]
[2020-02-12 11:33:16] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/styles.css 200 [0.03, 127.0.0.1:1832]
[2020-02-12 11:33:22] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/imgback.jpg 200 [0.09, 127.0.0.1:1832]
[2020-02-12 11:33:23] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-12 11:33:23] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.04, 127.0.0.1:1832]
[2020-02-12 11:34:05] INFO [django.channels.server.log_action:147] HTTP POST /login/ 302 [0.79, 127.0.0.1:1832]
[2020-02-12 11:34:05] INFO [django.channels.server.log_action:141] HTTP GET /courses/userinterest 200 [0.11, 127.0.0.1:1832]
[2020-02-12 11:34:05] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.02, 127.0.0.1:1832]
[2020-02-12 11:34:05] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.02, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/lin-logo.png 200 [0.02, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/button-left.png 200 [0.04, 127.0.0.1:1832]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/amazons.png 200 [0.02, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ppt.png 200 [0.01, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/jjava.png 200 [0.02, 127.0.0.1:1832]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/rchsaa.png 200 [0.02, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ajs.png 200 [0.01, 127.0.0.1:1832]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/csecurity.png 200 [0.01, 127.0.0.1:1832]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/biz.png 200 [0.01, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/dba.png 200 [0.02, 127.0.0.1:1862]
[2020-02-12 11:34:06] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/py.png 200 [0.03, 127.0.0.1:1832]
[2020-02-12 11:34:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs//list-bg.png 200 [0.07, 127.0.0.1:1832]
[2020-02-12 11:34:07] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.03, 127.0.0.1:1832]
[2020-02-12 11:34:16] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/ 302 [0.05, 127.0.0.1:1832]
[2020-02-12 11:34:17] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/ 302 [0.04, 127.0.0.1:1862]
[2020-02-12 11:35:18] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [61.07, 127.0.0.1:1862]
[2020-02-12 11:35:20] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/IMG-20191210-WA0011_2.jpg 200 [0.23, 127.0.0.1:1862]
[2020-02-12 11:36:30] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/images/ui-icons_444444_256x240.png 200 [0.06, 127.0.0.1:2046]
[2020-02-12 11:36:30] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_OxuxiCD.pdf 200 [0.08, 127.0.0.1:2047]
[2020-02-12 11:41:42] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [40.35, 127.0.0.1:2097]
[2020-02-12 11:41:52] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_miy0JHV.pdf 200 [0.09, 127.0.0.1:2097]
[2020-02-12 11:42:45] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [44.01, 127.0.0.1:2097]
[2020-02-12 11:42:49] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.08, 127.0.0.1:2097]
[2020-02-12 11:42:58] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_dtEsMEV.pdf 200 [0.10, 127.0.0.1:2097]
[2020-02-12 11:44:01] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [34.35, 127.0.0.1:2097]
[2020-02-12 11:44:09] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_whbvOhz.pdf 200 [0.06, 127.0.0.1:2097]
[2020-02-12 11:44:49] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [33.03, 127.0.0.1:2097]
[2020-02-12 11:44:52] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.03, 127.0.0.1:2097]
[2020-02-12 11:47:29] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_x6TAz8I.pdf 200 [0.03, 127.0.0.1:2428]
[2020-02-12 11:52:52] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [38.44, 127.0.0.1:2470]
[2020-02-12 11:52:57] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Terms-mololuwa_olufeko_LYV6vNH.pdf 200 [0.04, 127.0.0.1:2470]
[2020-02-12 11:53:02] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_oPiquA0.pdf 200 [0.06, 127.0.0.1:2470]
[2020-02-12 11:53:09] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_nTCXSCN.pdf 200 [0.04, 127.0.0.1:2470]
[2020-02-12 11:54:03] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [40.88, 127.0.0.1:2572]
[2020-02-12 11:54:06] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.01, 127.0.0.1:2572]
[2020-02-12 11:54:09] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_A4NYD9x.pdf 200 [0.07, 127.0.0.1:2572]
[2020-02-12 11:54:14] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_jxjmgna.pdf 200 [0.04, 127.0.0.1:2572]
[2020-02-12 11:56:01] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [31.97, 127.0.0.1:2842]
[2020-02-12 11:56:28] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/profile/ 302 [0.08, 127.0.0.1:2842]
[2020-02-12 11:57:07] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [38.60, 127.0.0.1:2842]
[2020-02-12 11:57:07] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/1.jpg 200 [0.07, 127.0.0.1:2842]
[2020-02-12 11:57:20] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_thAqRbs.pdf 200 [0.43, 127.0.0.1:2842]
[2020-02-12 11:57:27] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_oCwaZvj.pdf 200 [0.04, 127.0.0.1:2842]
[2020-02-12 11:57:33] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Terms-mololuwa_olufeko_79DtTJF.pdf 200 [0.08, 127.0.0.1:2842]
[2020-02-12 12:36:08] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/profile/ 302 [0.20, 127.0.0.1:1602]
[2020-02-12 12:36:08] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.16, 127.0.0.1:1602]
[2020-02-12 12:36:10] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/proj-list.png 200 [0.09, 127.0.0.1:1602]
[2020-02-12 12:36:13] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.22, 127.0.0.1:1627]
[2020-02-12 12:36:52] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [38.30, 127.0.0.1:1627]
[2020-02-12 12:36:52] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/1.jpg 200 [0.14, 127.0.0.1:1777]
[2020-02-12 14:13:16] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/profile/ 302 [0.28, 127.0.0.1:31816]
[2020-02-12 14:13:16] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.27, 127.0.0.1:31816]
[2020-02-12 14:13:22] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.22, 127.0.0.1:31838]
[2020-02-12 14:14:18] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [55.56, 127.0.0.1:31838]
[2020-02-12 14:14:18] INFO [django.channels.server.log_action:145] HTTP GET /media/resume/1.jpg 304 [0.15, 127.0.0.1:31838]
[2020-02-12 14:15:40] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_EscZn4f.pdf 200 [0.10, 127.0.0.1:31947]
[2020-02-12 14:15:46] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_Vu8wFoU.pdf 200 [0.14, 127.0.0.1:31947]
[2020-02-12 14:17:10] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [27.97, 127.0.0.1:31958]
[2020-02-12 14:17:33] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_qq5NWMz.pdf 200 [0.13, 127.0.0.1:32046]
[2020-02-12 14:17:40] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_zC2LQRk.pdf 200 [0.13, 127.0.0.1:32054]
[2020-02-12 14:17:48] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Terms-mololuwa_olufeko_m1P3nrG.pdf 200 [0.04, 127.0.0.1:32056]
[2020-02-12 14:18:40] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [26.57, 127.0.0.1:32060]
[2020-02-12 14:18:44] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.08, 127.0.0.1:32060]
[2020-02-12 14:19:34] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_DzLCveO.pdf 200 [0.40, 127.0.0.1:32175]
[2020-02-12 14:19:40] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_LoOBIBm.pdf 200 [0.08, 127.0.0.1:32178]
[2020-02-12 14:19:52] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Terms-mololuwa_olufeko_gynALJZ.pdf 200 [0.09, 127.0.0.1:32180]
[2020-02-12 14:20:50] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [42.42, 127.0.0.1:32184]
[2020-02-12 14:20:55] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.06, 127.0.0.1:32184]
[2020-02-12 14:21:02] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-mololuwa_olufeko_PYCjyZM.pdf 200 [0.12, 127.0.0.1:32274]
[2020-02-12 14:21:04] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Terms-mololuwa_olufeko_RAxo4ba.pdf 200 [0.08, 127.0.0.1:32274]
[2020-02-12 14:21:11] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_tbSyWlm.pdf 200 [0.04, 127.0.0.1:32279]
[2020-02-13 17:32:52] INFO [django.channels.server.log_action:147] HTTP GET / 302 [0.86, 127.0.0.1:27206]
[2020-02-13 17:32:52] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.27, 127.0.0.1:27206]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.10, 127.0.0.1:27206]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.09, 127.0.0.1:27207]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.10, 127.0.0.1:27231]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.10, 127.0.0.1:27233]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.12, 127.0.0.1:27234]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.11, 127.0.0.1:27207]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.11, 127.0.0.1:27206]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.18, 127.0.0.1:27232]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.12, 127.0.0.1:27233]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.08, 127.0.0.1:27234]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.07, 127.0.0.1:27232]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.09, 127.0.0.1:27207]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.10, 127.0.0.1:27206]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.20, 127.0.0.1:27231]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.04, 127.0.0.1:27207]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.06, 127.0.0.1:27232]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.03, 127.0.0.1:27232]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.04, 127.0.0.1:27207]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.02, 127.0.0.1:27232]
[2020-02-13 17:32:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/proj-list.png 200 [0.05, 127.0.0.1:27232]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.19, 127.0.0.1:27232]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.17, 127.0.0.1:27232]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.04, 127.0.0.1:27232]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.04, 127.0.0.1:27207]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.05, 127.0.0.1:27231]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.04, 127.0.0.1:27231]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.07, 127.0.0.1:27207]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.03, 127.0.0.1:27231]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.02, 127.0.0.1:27207]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.02, 127.0.0.1:27207]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.01, 127.0.0.1:27207]
[2020-02-13 17:32:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.05, 127.0.0.1:27231]
[2020-02-13 17:33:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.05, 127.0.0.1:27231]
[2020-02-13 17:33:00] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.05, 127.0.0.1:27207]
[2020-02-13 17:33:01] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.09, 127.0.0.1:27251]
[2020-02-13 17:33:47] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [24.50, 127.0.0.1:27257]
[2020-02-13 17:33:48] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/1.jpg 200 [0.15, 127.0.0.1:27257]
[2020-02-13 17:34:34] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/images/ui-icons_444444_256x240.png 200 [0.09, 127.0.0.1:27365]
[2020-02-13 17:34:34] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-ISA-mololuwa_olufeko_JS0BgoP.pdf 200 [0.12, 127.0.0.1:27366]
[2020-02-13 17:34:49] INFO [django.channels.server.log_action:147] HTTP GET /admin 301 [0.01, 127.0.0.1:27375]
[2020-02-13 17:34:49] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.26, 127.0.0.1:27375]
[2020-02-13 17:34:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/dashboard.css 200 [0.10, 127.0.0.1:27376]
[2020-02-13 17:34:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/base.css 200 [0.11, 127.0.0.1:27375]
[2020-02-13 17:34:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/responsive.css 200 [0.11, 127.0.0.1:27384]
[2020-02-13 17:34:50] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.20, 127.0.0.1:27376]
[2020-02-13 17:34:50] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/fonts.css 200 [0.01, 127.0.0.1:27376]
[2020-02-13 17:34:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-addlink.svg 200 [0.08, 127.0.0.1:27398]
[2020-02-13 17:34:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-changelink.svg 200 [0.08, 127.0.0.1:27399]
[2020-02-13 17:34:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Bold-webfont.woff 200 [0.05, 127.0.0.1:27398]
[2020-02-13 17:34:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Regular-webfont.woff 200 [0.06, 127.0.0.1:27401]
[2020-02-13 17:34:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Light-webfont.woff 200 [0.07, 127.0.0.1:27399]
[2020-02-13 17:34:52] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-13 17:34:52] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.05, 127.0.0.1:27399]
[2020-02-13 17:34:56] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.44, 127.0.0.1:27399]
[2020-02-13 17:34:56] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:27401]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/changelists.css 200 [0.09, 127.0.0.1:27399]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/jquery.init.js 200 [0.08, 127.0.0.1:27415]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/vendor/jquery/jquery.js 200 [0.12, 127.0.0.1:27398]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/actions.js 200 [0.09, 127.0.0.1:27401]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/core.js 200 [0.13, 127.0.0.1:27414]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/prepopulate.js 200 [0.04, 127.0.0.1:27415]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/urlify.js 200 [0.07, 127.0.0.1:27399]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/admin/RelatedObjectLookups.js 200 [0.12, 127.0.0.1:27416]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/vendor/xregexp/xregexp.js 200 [0.06, 127.0.0.1:27398]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/search.svg 200 [0.06, 127.0.0.1:27415]
[2020-02-13 17:34:57] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-no.svg 200 [0.06, 127.0.0.1:27399]
[2020-02-13 17:34:58] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/tooltag-add.svg 200 [0.01, 127.0.0.1:27399]
[2020-02-13 17:34:58] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/sorting-icons.svg 200 [0.04, 127.0.0.1:27415]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.43, 127.0.0.1:27421]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/forms.css 200 [0.08, 127.0.0.1:27421]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/change_form.js 200 [0.05, 127.0.0.1:27434]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/calendar.js 200 [0.10, 127.0.0.1:27430]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/admin/DateTimeShortcuts.js 200 [0.10, 127.0.0.1:27431]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.11, 127.0.0.1:27422]
[2020-02-13 17:35:00] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/prepopulate_init.js 200 [0.02, 127.0.0.1:27421]
[2020-02-13 17:35:01] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/widgets.css 200 [0.04, 127.0.0.1:27434]
[2020-02-13 17:35:01] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/assets/vendor/bootstrap/css/bootstrap.min.css 200 [0.13, 127.0.0.1:27432]
[2020-02-13 17:35:02] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-calendar.svg 200 [0.02, 127.0.0.1:27432]
[2020-02-13 17:35:02] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-clock.svg 200 [0.04, 127.0.0.1:27434]
[2020-02-13 17:35:38] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [25.88, 127.0.0.1:27447]
[2020-02-13 17:35:48] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-ISA-mololuwa_olufeko.pdf 200 [0.09, 127.0.0.1:27529]
[2020-02-13 17:35:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.46, 127.0.0.1:27533]
[2020-02-13 17:35:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:27533]
[2020-02-13 17:40:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.34, 127.0.0.1:27619]
[2020-02-13 17:40:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:27619]
[2020-02-13 17:40:20] INFO [django.channels.server.log_action:141] HTTP GET /media/uploads/Work-Experience-Eligibility-Samuel_Olufeko.pdf 200 [0.12, 127.0.0.1:27631]
[2020-02-13 17:40:24] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.38, 127.0.0.1:27631]
[2020-02-13 17:41:03] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [24.05, 127.0.0.1:27653]
[2020-02-13 17:43:10] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.45, 127.0.0.1:27843]
[2020-02-13 17:43:10] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:27843]
[2020-02-13 17:43:18] INFO [django.channels.server.log_action:141] HTTP GET /logout 200 [0.16, 127.0.0.1:27860]
[2020-02-13 17:43:20] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/slide1.png 200 [0.12, 127.0.0.1:27872]
[2020-02-13 17:43:22] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.13, 127.0.0.1:27872]
[2020-02-13 17:43:25] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.01, 127.0.0.1:27872]
[2020-02-13 17:43:25] INFO [django.channels.server.log_action:141] HTTP GET /signup 200 [0.04, 127.0.0.1:27872]
[2020-02-13 17:43:25] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/styles.css 200 [0.04, 127.0.0.1:27872]
[2020-02-13 17:43:27] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/imgback.jpg 200 [0.09, 127.0.0.1:27872]
[2020-02-13 17:43:51] INFO [django.channels.server.log_action:141] HTTP POST /signup 200 [0.46, 127.0.0.1:27910]
[2020-02-13 17:43:51] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/checkers.png 200 [0.06, 127.0.0.1:27910]
[2020-02-13 17:43:53] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/signupbg.png 200 [0.07, 127.0.0.1:27910]
[2020-02-13 17:43:55] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.11, 127.0.0.1:27910]
[2020-02-13 17:43:58] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.04, 127.0.0.1:27910]
[2020-02-13 17:44:01] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/ 200 [0.16, 127.0.0.1:27940]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/we_new.css 200 [0.05, 127.0.0.1:27940]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/arrow-right.png 200 [0.05, 127.0.0.1:27947]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/workexperience.css 200 [0.06, 127.0.0.1:27941]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/cup.png 200 [0.07, 127.0.0.1:27948]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/tap.png 200 [0.04, 127.0.0.1:27948]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/accept.png 200 [0.05, 127.0.0.1:27941]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/number.png 200 [0.03, 127.0.0.1:27948]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/join.png 200 [0.03, 127.0.0.1:27941]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/arrow-blue.png 200 [0.01, 127.0.0.1:27948]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/graph.svg 200 [0.03, 127.0.0.1:27941]
[2020-02-13 17:44:02] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/list%20_checked.png 200 [0.02, 127.0.0.1:27948]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/cap.png 200 [0.07, 127.0.0.1:27948]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/wallet.png 200 [0.04, 127.0.0.1:27948]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/el.png 200 [0.02, 127.0.0.1:27941]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/bg2.png 200 [0.03, 127.0.0.1:27948]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/alumni-bg.png 200 [0.05, 127.0.0.1:27947]
[2020-02-13 17:44:03] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/thmbs.png 200 [0.05, 127.0.0.1:27941]
[2020-02-13 17:44:08] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/terms/ 200 [0.16, 127.0.0.1:27941]
[2020-02-13 17:44:12] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/pay/ 200 [0.20, 127.0.0.1:27984]
[2020-02-13 17:44:19] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/linuxlogo.png 200 [0.05, 127.0.0.1:27984]
[2020-02-13 17:44:36] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/pay/ 302 [1.59, 127.0.0.1:28013]
[2020-02-13 17:44:36] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/eligibility/ 200 [0.20, 127.0.0.1:28013]
[2020-02-13 17:44:57] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/eligibility/ 302 [5.02, 127.0.0.1:28035]
[2020-02-13 17:44:57] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/ISA/ 200 [0.22, 127.0.0.1:28035]
[2020-02-13 17:45:11] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/ISA/ 302 [0.13, 127.0.0.1:28070]
[2020-02-13 17:45:12] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/sign/ 200 [0.18, 127.0.0.1:28070]
[2020-02-13 17:45:37] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/sign/ 302 [20.19, 127.0.0.1:28070]
[2020-02-13 17:45:37] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/apply/ 200 [0.67, 127.0.0.1:28070]
[2020-02-13 17:46:16] INFO [django.channels.server.log_action:147] HTTP POST /workexperience/apply/ 302 [0.05, 127.0.0.1:28188]
[2020-02-13 17:46:43] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [26.26, 127.0.0.1:28188]
[2020-02-13 17:46:43] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/prof1.png 200 [0.05, 127.0.0.1:28188]
[2020-02-13 17:46:43] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/prof2.png 200 [0.01, 127.0.0.1:28188]
[2020-02-13 17:47:40] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-ISA-owode_tes.pdf 200 [0.11, 127.0.0.1:28303]
[2020-02-13 17:47:49] INFO [django.channels.server.log_action:147] HTTP GET /admin/home/workexperienceeligibility/ 302 [0.04, 127.0.0.1:28303]
[2020-02-13 17:47:49] INFO [django.channels.server.log_action:141] HTTP GET /admin/login/?next=/admin/home/workexperienceeligibility/ 200 [0.08, 127.0.0.1:28303]
[2020-02-13 17:47:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/login.css 200 [0.06, 127.0.0.1:28303]
[2020-02-13 17:47:56] INFO [django.channels.server.log_action:147] HTTP POST /admin/login/?next=/admin/home/workexperienceeligibility/ 302 [0.24, 127.0.0.1:28326]
[2020-02-13 17:47:57] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.38, 127.0.0.1:28326]
[2020-02-13 17:47:57] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.03, 127.0.0.1:28326]
[2020-02-13 17:48:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.34, 127.0.0.1:28326]
[2020-02-13 17:48:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:28326]
[2020-02-13 20:39:34] INFO [django.channels.server.log_action:147] HTTP GET /workexperience/profile/ 302 [4.06, 127.0.0.1:21800]
[2020-02-13 20:39:35] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [1.19, 127.0.0.1:21800]
[2020-02-13 21:04:52] INFO [django.channels.server.log_action:147] HTTP GET /admin/ 302 [0.64, 127.0.0.1:22604]
[2020-02-13 21:04:52] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.59, 127.0.0.1:22604]
[2020-02-13 21:05:31] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [1.01, 127.0.0.1:22648]
[2020-02-13 21:06:54] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [83.25, 127.0.0.1:22648]
[2020-02-13 21:06:55] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/1.jpg 200 [0.28, 127.0.0.1:22648]
[2020-02-13 21:07:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.75, 127.0.0.1:22911]
[2020-02-13 21:07:52] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.34, 127.0.0.1:22911]
[2020-02-13 21:08:11] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.60, 127.0.0.1:22943]
[2020-02-13 21:08:12] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.10, 127.0.0.1:22943]
[2020-02-13 21:08:22] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.55, 127.0.0.1:22957]
[2020-02-13 21:08:22] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:22957]
[2020-02-14 18:59:06] INFO [django.channels.server.log_action:141] HTTP GET / 200 [0.63, 127.0.0.1:2061]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.04, 127.0.0.1:2061]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.05, 127.0.0.1:2062]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.04, 127.0.0.1:2067]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.05, 127.0.0.1:2061]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.08, 127.0.0.1:2070]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.06, 127.0.0.1:2067]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.12, 127.0.0.1:2069]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.05, 127.0.0.1:2061]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.11, 127.0.0.1:2062]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.15, 127.0.0.1:2068]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/testimonial-timeline.css 200 [0.06, 127.0.0.1:2070]
[2020-02-14 18:59:07] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.07, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.04, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/LaptopBook.png 200 [0.05, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.02, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.03, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.03, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.07, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobphone.png 200 [0.05, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homelive.png 200 [0.05, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get3.png 200 [0.06, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get1.png 200 [0.01, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/get2.png 200 [0.03, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/kenna.png 200 [0.03, 127.0.0.1:2067]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/adekunle.png 200 [0.04, 127.0.0.1:2070]
[2020-02-14 18:59:17] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/fetigue.png 200 [0.02, 127.0.0.1:2067]
[2020-02-14 18:59:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.04, 127.0.0.1:2067]
[2020-02-14 18:59:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/jobban2.png 200 [0.05, 127.0.0.1:2067]
[2020-02-14 18:59:21] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/homebanner.png 200 [0.06, 127.0.0.1:2070]
[2020-02-14 18:59:35] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.01, 127.0.0.1:2070]
[2020-02-14 21:13:20] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [3.03, 127.0.0.1:2472]
[2020-02-14 21:28:50] INFO [django.channels.server.log_action:141] HTTP GET /login/ 200 [0.85, 127.0.0.1:3108]
[2020-02-14 21:28:50] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/styles.css 200 [0.03, 127.0.0.1:3108]
[2020-02-14 21:28:51] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/imgback.jpg 200 [0.11, 127.0.0.1:3124]
[2020-02-14 21:28:51] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-14 21:28:51] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.03, 127.0.0.1:3124]
[2020-02-14 21:28:57] INFO [django.channels.server.log_action:147] HTTP POST /login/ 302 [0.76, 127.0.0.1:3124]
[2020-02-14 21:28:57] INFO [django.channels.server.log_action:141] HTTP GET /courses/userinterest 200 [0.18, 127.0.0.1:3124]
[2020-02-14 21:28:57] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.06, 127.0.0.1:3124]
[2020-02-14 21:28:57] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.04, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/lin-logo.png 200 [0.07, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/button-left.png 200 [0.08, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/amazons.png 200 [0.04, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ppt.png 200 [0.05, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/jjava.png 200 [0.05, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/rchsaa.png 200 [0.04, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/ajs.png 200 [0.04, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/csecurity.png 200 [0.04, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/dba.png 200 [0.02, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/biz.png 200 [0.05, 127.0.0.1:3127]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/py.png 200 [0.04, 127.0.0.1:3124]
[2020-02-14 21:28:58] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs//list-bg.png 200 [0.10, 127.0.0.1:3124]
[2020-02-14 21:29:00] INFO [django.channels.server.log_action:141] HTTP POST /ulocation 200 [0.08, 127.0.0.1:3124]
[2020-02-14 21:30:04] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [62.25, 127.0.0.1:3152]
[2020-02-14 21:30:04] INFO [django.channels.server.log_action:141] HTTP GET /media/resume/1.jpg 200 [0.11, 127.0.0.1:3152]
[2020-02-14 21:33:57] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [63.61, 127.0.0.1:3293]
[2020-02-14 21:37:12] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/images/ui-icons_444444_256x240.png 200 [0.36, 127.0.0.1:1040]
[2020-02-14 21:37:12] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-Eligibility-mololuwa_olufeko_oLeDHrW.pdf 200 [0.34, 127.0.0.1:1041]
[2020-02-14 21:38:05] INFO [django.channels.server.log_action:141] HTTP GET /workexperience/profile/ 200 [40.99, 127.0.0.1:1046]
[2020-02-14 21:38:10] INFO [django.channels.server.log_action:141] HTTP GET /static/favicon.ico 200 [0.10, 127.0.0.1:1046]
[2020-02-14 21:43:10] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-Eligibility-mololuwa_olufeko_OEFsCd6.pdf 200 [0.29, 127.0.0.1:1419]
[2020-02-14 21:43:23] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-ISA-mololuwa_olufeko_HtYLntO.pdf 200 [0.12, 127.0.0.1:1419]
[2020-02-14 21:43:31] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-Terms-mololuwa_olufeko_qLFWGEq.pdf 200 [0.11, 127.0.0.1:1419]
[2020-02-16 21:51:40] INFO [django.channels.server.log_action:147] HTTP GET /admin 301 [0.26, 127.0.0.1:3255]
[2020-02-16 21:51:40] INFO [django.channels.server.log_action:147] HTTP GET /robots.txt 302 [0.31, 127.0.0.1:3256]
[2020-02-16 21:51:40] INFO [django.channels.server.log_action:147] HTTP GET /admin/ 302 [0.29, 127.0.0.1:3255]
[2020-02-16 21:51:40] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.25, 127.0.0.1:3256]
[2020-02-16 21:51:40] INFO [django.channels.server.log_action:141] HTTP GET /timeout 200 [0.08, 127.0.0.1:3255]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validate.js 200 [0.10, 127.0.0.1:3256]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/validation/jquery.validation.functions.js 200 [0.11, 127.0.0.1:3255]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/bootstrap.min.css 200 [0.12, 127.0.0.1:3269]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/css/style.css 200 [0.13, 127.0.0.1:3271]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/main.css 200 [0.07, 127.0.0.1:3255]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/style.css 200 [0.15, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstyle2.css 200 [0.18, 127.0.0.1:3272]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/css/newstylenew.css 200 [0.10, 127.0.0.1:3256]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/fonts/fontsstylesheet.css 200 [0.06, 127.0.0.1:3255]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.css 200 [0.08, 127.0.0.1:3271]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/jquery-ui-1.12.1.custom/jquery-ui.min.js 200 [0.07, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/Projects/custom/css/projects.css 200 [0.15, 127.0.0.1:3269]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/icon1.png 200 [0.06, 127.0.0.1:3271]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logo.png 200 [0.07, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newdesigner/js/bootstrap.min.js 200 [0.04, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/jquery.mobile.custom.min.js 200 [0.04, 127.0.0.1:3271]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/faq/js/main.js 200 [0.03, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/home/images/logout1.png 200 [0.01, 127.0.0.1:3270]
[2020-02-16 21:51:41] INFO [django.channels.server.log_action:141] HTTP GET /static/js/waterbubble.js 200 [0.04, 127.0.0.1:3271]
[2020-02-16 21:51:42] INFO [django.channels.server.log_action:141] HTTP GET /static/home/css/newhomepage/imgs/proj-list.png 200 [0.07, 127.0.0.1:3271]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:147] HTTP POST /timeout 302 [0.23, 127.0.0.1:3286]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:141] HTTP GET /admin/ 200 [0.30, 127.0.0.1:3286]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/dashboard.css 200 [0.02, 127.0.0.1:3287]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/responsive.css 200 [0.05, 127.0.0.1:3295]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/base.css 200 [0.07, 127.0.0.1:3286]
[2020-02-16 21:51:47] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/fonts.css 200 [0.02, 127.0.0.1:3286]
[2020-02-16 21:51:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-addlink.svg 200 [0.01, 127.0.0.1:3286]
[2020-02-16 21:51:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-changelink.svg 200 [0.02, 127.0.0.1:3295]
[2020-02-16 21:51:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Light-webfont.woff 200 [0.06, 127.0.0.1:3295]
[2020-02-16 21:51:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Regular-webfont.woff 200 [0.06, 127.0.0.1:3287]
[2020-02-16 21:51:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/fonts/Roboto-Bold-webfont.woff 200 [0.06, 127.0.0.1:3286]
[2020-02-16 21:51:49] WARNING [django.request.get_response:93] Not Found: /favicon.ico
[2020-02-16 21:51:49] WARNING [django.channels.server.log_action:149] HTTP GET /favicon.ico 404 [0.07, 127.0.0.1:3286]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.68, 127.0.0.1:3312]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/jquery.init.js 200 [0.05, 127.0.0.1:3320]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/changelists.css 200 [0.08, 127.0.0.1:3312]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3313]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/admin/RelatedObjectLookups.js 200 [0.07, 127.0.0.1:3322]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/core.js 200 [0.08, 127.0.0.1:3321]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/actions.js 200 [0.04, 127.0.0.1:3320]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/vendor/jquery/jquery.js 200 [0.11, 127.0.0.1:3319]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/urlify.js 200 [0.05, 127.0.0.1:3312]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/prepopulate.js 200 [0.03, 127.0.0.1:3322]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/vendor/xregexp/xregexp.js 200 [0.03, 127.0.0.1:3321]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/search.svg 200 [0.02, 127.0.0.1:3322]
[2020-02-16 21:52:48] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-no.svg 200 [0.03, 127.0.0.1:3319]
[2020-02-16 21:52:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/sorting-icons.svg 200 [0.01, 127.0.0.1:3322]
[2020-02-16 21:52:49] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/tooltag-add.svg 200 [0.06, 127.0.0.1:3319]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.43, 127.0.0.1:3319]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/forms.css 200 [0.05, 127.0.0.1:3319]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/calendar.js 200 [0.06, 127.0.0.1:3322]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/admin/DateTimeShortcuts.js 200 [0.05, 127.0.0.1:3312]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/assets/vendor/bootstrap/css/bootstrap.min.css 200 [0.07, 127.0.0.1:3320]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/css/widgets.css 200 [0.02, 127.0.0.1:3322]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3321]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/prepopulate_init.js 200 [0.01, 127.0.0.1:3322]
[2020-02-16 21:52:52] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/js/change_form.js 200 [0.03, 127.0.0.1:3321]
[2020-02-16 21:52:54] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-clock.svg 200 [0.02, 127.0.0.1:3344]
[2020-02-16 21:52:54] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-calendar.svg 200 [0.02, 127.0.0.1:3343]
[2020-02-16 21:52:58] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/70/change/ 200 [0.42, 127.0.0.1:3343]
[2020-02-16 21:52:58] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.07, 127.0.0.1:3343]
[2020-02-16 21:53:19] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/70/change/ 200 [0.36, 127.0.0.1:3359]
[2020-02-16 21:53:19] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3359]
[2020-02-16 21:53:32] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/70/change/ 302 [0.11, 127.0.0.1:3374]
[2020-02-16 21:53:32] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.33, 127.0.0.1:3374]
[2020-02-16 21:53:33] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.11, 127.0.0.1:3374]
[2020-02-16 21:53:34] INFO [django.channels.server.log_action:141] HTTP GET /static/admin/img/icon-yes.svg 200 [0.02, 127.0.0.1:3386]
[2020-02-16 21:53:46] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.35, 127.0.0.1:3399]
[2020-02-16 21:53:46] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:3399]
[2020-02-16 21:54:06] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/70/change/ 200 [0.37, 127.0.0.1:3418]
[2020-02-16 21:54:06] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3418]
[2020-02-16 21:55:37] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/70/change/ 302 [0.39, 127.0.0.1:3488]
[2020-02-16 21:55:38] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.37, 127.0.0.1:3488]
[2020-02-16 21:55:38] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:3488]
[2020-02-16 21:58:37] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.81, 127.0.0.1:3560]
[2020-02-16 21:58:37] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3560]
[2020-02-16 21:58:47] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/70/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 619, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 835, in save_model
    print(obj.message.slug)
AttributeError: 'WorkExperienceEligibility' object has no attribute 'message'
[2020-02-16 21:58:47] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/70/change/ 500 [0.13, 127.0.0.1:3581]
[2020-02-16 21:59:31] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/70/change/ 302 [0.45, 127.0.0.1:3618]
[2020-02-16 21:59:31] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.35, 127.0.0.1:3618]
[2020-02-16 21:59:31] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3618]
[2020-02-16 21:59:35] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.34, 127.0.0.1:3631]
[2020-02-16 21:59:35] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3631]
[2020-02-16 21:59:42] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/70/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 619, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 824, in save_model
    html_template = get_template('home/workexpeligibilitypdf.html').render({'details':details,'date':date,'ssn':ssn,'created':created})
NameError: name 'get_template' is not defined
[2020-02-16 21:59:42] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/70/change/ 500 [0.12, 127.0.0.1:3631]
[2020-02-16 22:00:27] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/70/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 620, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 828, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
NameError: name 'HTML' is not defined
[2020-02-16 22:00:27] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/70/change/ 500 [0.36, 127.0.0.1:3661]
[2020-02-16 22:01:10] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/70/change/ 302 [6.68, 127.0.0.1:3693]
[2020-02-16 22:01:10] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.34, 127.0.0.1:3693]
[2020-02-16 22:01:10] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.09, 127.0.0.1:3693]
[2020-02-16 22:02:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.36, 127.0.0.1:3722]
[2020-02-16 22:02:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3722]
[2020-02-16 22:02:21] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/69/change/ 200 [0.44, 127.0.0.1:3722]
[2020-02-16 22:02:21] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:3722]
[2020-02-16 22:03:28] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/69/change/ 200 [0.70, 127.0.0.1:3795]
[2020-02-16 22:03:28] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:3795]
[2020-02-16 22:04:43] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/69/change/ 200 [0.67, 127.0.0.1:3868]
[2020-02-16 22:04:43] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.08, 127.0.0.1:3868]
[2020-02-16 22:05:41] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/69/change/ 200 [0.67, 127.0.0.1:3941]
[2020-02-16 22:05:41] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3941]
[2020-02-16 22:05:53] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/69/change/ 302 [5.14, 127.0.0.1:3941]
[2020-02-16 22:05:54] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.34, 127.0.0.1:3941]
[2020-02-16 22:05:54] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3941]
[2020-02-16 22:06:12] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/59/change/ 200 [0.35, 127.0.0.1:3984]
[2020-02-16 22:06:12] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.07, 127.0.0.1:3984]
[2020-02-16 22:06:21] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceeligibility/59/change/ 200 [0.34, 127.0.0.1:3997]
[2020-02-16 22:06:21] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:3997]
[2020-02-16 22:07:17] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/59/change/ 302 [2.81, 127.0.0.1:4050]
[2020-02-16 22:07:18] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.34, 127.0.0.1:4050]
[2020-02-16 22:07:18] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4050]
[2020-02-16 22:07:49] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.35, 127.0.0.1:4076]
[2020-02-16 22:07:49] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4076]
[2020-02-16 22:07:59] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/62/change/ 302 [3.25, 127.0.0.1:4088]
[2020-02-16 22:08:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.40, 127.0.0.1:4088]
[2020-02-16 22:08:00] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4088]
[2020-02-16 22:08:05] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.37, 127.0.0.1:4109]
[2020-02-16 22:08:05] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:4109]
[2020-02-16 22:08:50] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/62/change/ 302 [3.73, 127.0.0.1:4124]
[2020-02-16 22:08:50] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.31, 127.0.0.1:4124]
[2020-02-16 22:08:50] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:4124]
[2020-02-16 22:08:56] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.33, 127.0.0.1:4148]
[2020-02-16 22:08:56] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4148]
[2020-02-16 22:10:19] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/62/change/ 302 [7.69, 127.0.0.1:4191]
[2020-02-16 22:10:19] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.36, 127.0.0.1:4191]
[2020-02-16 22:10:19] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4191]
[2020-02-16 22:10:26] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.33, 127.0.0.1:4218]
[2020-02-16 22:10:26] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4218]
[2020-02-16 22:11:56] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/62/change/ 302 [2.88, 127.0.0.1:4260]
[2020-02-16 22:11:57] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.34, 127.0.0.1:4260]
[2020-02-16 22:11:57] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.06, 127.0.0.1:4260]
[2020-02-16 22:12:02] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.34, 127.0.0.1:4260]
[2020-02-16 22:12:02] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:4260]
[2020-02-16 22:12:16] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-Eligibility-Karim_Tamara_v61yD5U.pdf 200 [0.04, 127.0.0.1:4291]
[2020-02-16 22:12:21] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.39, 127.0.0.1:4291]
[2020-02-16 22:31:55] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:31:55] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [0.39, 127.0.0.1:1077]
[2020-02-16 22:34:44] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.52, 127.0.0.1:1108]
[2020-02-16 22:34:44] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.11, 127.0.0.1:1108]
[2020-02-16 22:35:01] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:35:01] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [12.23, 127.0.0.1:1108]
[2020-02-16 22:35:41] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:35:41] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [12.17, 127.0.0.1:1139]
[2020-02-16 22:36:11] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:36:11] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [12.18, 127.0.0.1:1172]
[2020-02-16 22:36:34] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:36:34] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [12.14, 127.0.0.1:1197]
[2020-02-16 22:36:49] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceeligibility/62/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1016, in _send_output
    self.send(msg)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 956, in send
    self.connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 1384, in connect
    super().connect()
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\http\client.py", line 928, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 284, in fetch
    result = url_fetcher(url)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 248, in default_url_fetcher
    timeout=timeout, context=ssl_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\urllib\request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 622, in change_view
    request, object_id, form_url, extra_context=extra_context,
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 831, in save_model
    pdf_file = HTML(string=html_template).write_pdf( stylesheets=[CSS("https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")],presentational_hints=True)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 322, in __init__
    with result as (source_type, source, base_url, protocol_encoding):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 396, in _select_source
    with result as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\__init__.py", line 406, in _select_source
    with fetch(url_fetcher, url) as result:
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\weasyprint\urls.py", line 286, in fetch
    raise URLFetchingError('%s: %s' % (type(exc).__name__, str(exc)))
weasyprint.urls.URLFetchingError: URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2020-02-16 22:36:49] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceeligibility/62/change/ 500 [0.25, 127.0.0.1:1221]
[2020-02-16 22:38:50] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.34, 127.0.0.1:1255]
[2020-02-16 22:38:51] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1255]
[2020-02-16 22:44:12] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceeligibility/62/change/ 302 [5.25, 127.0.0.1:1361]
[2020-02-16 22:44:12] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.34, 127.0.0.1:1361]
[2020-02-16 22:44:12] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1361]
[2020-02-16 22:47:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/62/change/ 200 [0.35, 127.0.0.1:1069]
[2020-02-16 22:47:14] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1069]
[2020-02-16 22:53:15] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/ 200 [0.64, 127.0.0.1:1177]
[2020-02-16 22:53:15] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.12, 127.0.0.1:1177]
[2020-02-16 22:53:23] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceeligibility/70/change/ 200 [0.35, 127.0.0.1:1177]
[2020-02-16 22:53:24] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:1177]
[2020-02-16 22:53:27] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/ 200 [0.12, 127.0.0.1:1208]
[2020-02-16 22:53:32] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceisa/ 200 [0.30, 127.0.0.1:1208]
[2020-02-16 22:53:32] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1208]
[2020-02-16 22:53:40] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceisa/25/change/ 200 [0.27, 127.0.0.1:1231]
[2020-02-16 22:53:40] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1231]
[2020-02-16 22:53:48] INFO [django.channels.server.log_action:141] HTTP POST /admin/home/workexperienceisa/25/change/ 200 [0.33, 127.0.0.1:1242]
[2020-02-16 22:53:48] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.15, 127.0.0.1:1242]
[2020-02-16 22:54:45] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceisa/25/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 609, in save_model
    details =  WorkExperienceEligibility.objects.get(user=user)
NameError: name 'user' is not defined
[2020-02-16 22:54:45] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceisa/25/change/ 500 [0.24, 127.0.0.1:1075]
[2020-02-16 22:55:03] ERROR [django.request.handle_uncaught_exception:118] Internal Server Error: /admin/home/workexperienceisa/25/change/
Traceback (most recent call last):
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 575, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1557, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1451, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\Sam\AppData\Local\Programs\Python\Python37\lib\site-packages\django\contrib\admin\options.py", line 1491, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "E:\Documents\Web\linux-contract\Django\Linuxjobber\home\admin.py", line 609, in save_model
    details =  WorkExperienceEligibility.objects.get(user=user)
NameError: name 'user' is not defined
[2020-02-16 22:55:03] ERROR [django.channels.server.log_action:154] HTTP POST /admin/home/workexperienceisa/25/change/ 500 [0.25, 127.0.0.1:1097]
[2020-02-16 22:57:33] INFO [django.channels.server.log_action:147] HTTP POST /admin/home/workexperienceisa/25/change/ 302 [129.63, 127.0.0.1:1121]
[2020-02-16 22:57:34] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceisa/ 200 [0.21, 127.0.0.1:1121]
[2020-02-16 22:57:34] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.05, 127.0.0.1:1193]
[2020-02-16 22:57:39] INFO [django.channels.server.log_action:141] HTTP GET /admin/home/workexperienceisa/25/change/ 200 [0.27, 127.0.0.1:1193]
[2020-02-16 22:57:39] INFO [django.channels.server.log_action:141] HTTP GET /admin/jsi18n/ 200 [0.04, 127.0.0.1:1193]
[2020-02-16 22:57:43] INFO [django.channels.server.log_action:141] HTTP GET /media/pdfs/Work-Experience-ISA-Karim_Tamara.pdf 200 [0.05, 127.0.0.1:1193]
